srcpkgs/pinebookpro-uboot/files/fcommon-Makefile.patch

```diff
@@ -1,20 +0,0 @@
---- pbp-uboot/Makefileg	2020-09-12 15:05:30.139856893 +0200
-+++ pbp-uboot/Makefile	2020-09-12 15:06:45.963853869 +0200
-@@ -273,7 +273,7 @@
```

Comment:
@pullmoll unfortunately this gcc10 patch did not apply cleanly to u-boot 2020.10.

Reply:
Just push as you like I'll take care of fixing for gcc10 again in case it is still required.